.github/workflows/ci.yml

```diff
@@ -1,10 +1,12 @@
 name: CI
 
+permissions:
+    contents: read
```

Comment:
Any reason for this change btw?

Reply:
Good question, others seem to ask this too: https://github.com/zumba/json-serializer/pull/57#discussion_r1321159944


It's not required, but if missing the permissions given to the CI token are wider and more insecure if someone manages to steal the token

https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

Reply:
Okay, I will revert this